README.md

```diff
@@ -693,6 +693,8 @@ maintaining the Node.js project.
   **Qingyu Deng** <<i@ayase-lab.com>>
 * [bmuenzenmeyer](https://github.com/bmuenzenmeyer) -
   **Brian Muenzenmeyer** <<brian.muenzenmeyer@gmail.com>> (he/him)
+* [atlowChemi](https://github.com/atlowChemi) -
+  **Chemi Atlow** <<chemi@atlow.co.il>> (he/him)
```

Comment:
Because we've had folks use non-Latin characters in their names in the past, we keep entries like this in alphabetical order by GitHub username. This entry should be moved to line 692. Other than that, it looks good to me.

Reply:
Oh I actually tried ordering, and I got confused (I recently merged 2 Github accounts -chemiAtlow & atlowChemi) 😅